apps/builder/package.json

```diff
@@ -93,8 +93,6 @@
     "isbot": "^5.1.25",
     "lexical": "^0.21.0",
     "match-sorter": "^8.0.0",
-    "mdast-util-from-markdown": "^2.0.2",
```

Comment:
hmm? what did you replace it with?

Reply:
micromark generates html and we have html parser now